docs/tutorials/getting-started.md

```diff
@@ -0,0 +1,118 @@
+
+### Deploy the Content-cache-k8s charm
+
+Since Content-cache is meant to serve another charm as its cache, we'll use Hello-kubecon as an example.
+
+Deploy the charms:
+
+```bash
+juju deploy content-cache-k8s
+juju deploy hello-kubecon
+```
+
+To see the pod created by the Content-cache-k8s charm, run `kubectl get pods` on a namespace named for the Juju model you've deployed the Content-cache-k8s charm into. The output is similar to the following:
+
+```bash
+modeloperator-98b8cb7df-m6stx   1/1     Running   0          2m20s
+content-cache-k8s-0             3/3     Running   0          2m
+hello-kubecon-0                 0/2     Running   0          36s
+```
+
+Run [`juju status`](https://juju.is/docs/olm/juju-status) to see the current status of the deployment. In the Unit list, you can see that Content-cache-k8s is blocked:
+
+```bash
+content-cache-k8s/0*  blocked   idle   10.1.97.227         Required config(s) empty: backend, site
+hello-kubecon/0*      active    idle   10.1.97.193   
+```
+
+This means that Content-cache-k8s charm isn't integrated with Hello-kubecon yet.
+
+### Relate to the Hello-kubecon charm
+
+Provide integration between Content-cache-k8s and Hello-kubecon by running the following [`juju relate`](https://juju.is/docs/olm/juju-relate) command:
+
+```bash
+juju relate content-cache-k8s:ingress-proxy hello-kubecon
+```
+
+Run `juju status` to see that the message has changed:
+
+```bash
+content-cache-k8s/0*  active    idle   10.1.97.227         Ready
+hello-kubecon/0*      active    idle   10.1.97.193
+```
+
+Note: `ingress-proxy` is the name of the relation. This is needed because establishes that the two charms are compatible with each other.  You can run `juju info content-cache-k8s` to check what are the relation names provided by the Content-cache-k8s application.
+
+The deployment finishes when the status shows "Active".
+
+### Relate to Ingress by using NGINX Ingress Integrator
+
+The NGINX Ingress Integrator charm can deploy and manage external access to HTTP/HTTPS services in a Kubernetes cluster.
+
+If you want to make Content-cache-k8s charm available to external clients, you need to deploy the NGINX Ingress Integrator charm and integrate Content-cache-k8s with it.
+
+See more details in [Adding the Ingress Relation to a Charm](https://charmhub.io/nginx-ingress-integrator/docs/adding-ingress-relation).
+
+Deploy the charm NGINX Ingress Integrator:
+
+```bash
+juju deploy nginx-ingress-integrator
+```
+
+If your cluster has [RBAC](https://kubernetes.io/docs/reference/access-authn-authz/rbac/) enabled, you'll be prompted to run the following:
+
+```bash
+juju trust nginx-ingress-integrator --scope cluster
+```
+
+Run `juju status` to verify the deployment.
+
+Provide integration between Content-cache-k8s and NGINX Ingress Integrator:
+
+```bash
+juju relate content-cache-k8s:ingress nginx-ingress-integrator
```

Comment:
Do you need to specify the "ingress" relation here? Just wondering if this would work without that (if so, it's better to be simpler).